Duplicati/Library/Backend/OAuthHelper/JSONWebHelper.cs

```diff
@@ -515,6 +574,20 @@ public async Task<HttpWebResponse> GetResponseAsync(AsyncHttpRequest req, Cancel
 
                 return (HttpWebResponse)req.GetResponse();
             }
+            /*
+             * If a web exception then grab the error detail from the response stream
+             */
+            catch (WebException wex)
+            {
+                using (Stream exstream = wex.Response.GetResponseStream())
+                {
+                    using (StreamReader reader = new StreamReader(exstream))

```

Comment:
I don't think you can guarantee there to be a response stream present? If the error is "Network disconnected" there will be no response stream and the user will get an exception here.